client/components/thank-you/index.tsx

```diff
@@ -206,8 +206,7 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
 	return (
 		<ThankYouContainer className={ classNames( 'thank-you__container', containerClassName ) }>
 			<ThankYouHeader className={ classNames( 'thank-you__container-header', headerClassName ) }>
-				{ /* eslint-disable-next-line jsx-a11y/alt-text */ }
-				<img { ...thankYouImage } />
+				<img { ...{ ...thankYouImage, alt: String( thankYouImage.alt ) } } />
```

Comment:
Fun fact! everywhere you use `String()` you can also use string template literals

```ts
`${ thankYouImage.alt }`
```